storage/docs/containers-storage.conf.5.md

```diff
@@ -1,4 +1,6 @@
-# containers-storage.conf 5 "May 2017"
+% containers-storage.conf 5 Container Storage Configuration File
+% Dan Walsh
+% May 2017
```

Comment:
Why this change? All the other pages use the same syntax so why is only that changed here?